app/assets/locales/android_translatable_strings.txt

```diff
@@ -93,7 +93,7 @@ menu.print.detail=Print Case Detail
 menu.enable.privileges=Mobile Privileges
 menu.privilege.claim.disable=Disable Privileges
 
-sync.success.sent.singular=Form sent to server!
+sync.success.sent.singular=${0} form sent to server!

```

Comment:
Hey, I'm confused as to why anything needed to change here. The idea previously was that `sync.success.sent.singular` should be used if only 1 form was sent to the server (in which case no parameter for # of forms is needed since it's always just 1), and then `sync.success.sent` should be used if more than 1 form was sent.